.github/workflows/package_for_release.yml

```diff
@@ -127,3 +132,36 @@ jobs:
               with:
                 name: live_compositor_with_web_renderer_darwin_aarch64.tar.gz
                 path: live_compositor_with_web_renderer_darwin_aarch64.tar.gz
+    docker:
+      runs-on: ubuntu-latest
+      steps:
+          - name: 📥 Checkout repo
+            uses: actions/checkout@v4
+
+          - name: 🔑 Log in to GitHub Container Registry
+            run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+          - name: 🔨 Build Live Compositor image
+            run: |
+              docker build -t ghcr.io/software-mansion/live-compositor:${{ inputs.version }} -f build_tools/docker/slim.Dockerfile .
```

Comment:
can you run this image on macOS or do we need a separate arm version?

Reply:
It works via x64 emulation

Reply:
I tried running web renderer variant and on x64 it just hangs and doesn't start at all. On arm64 it works